wagtail/admin/templates/wagtailadmin/skeleton.html

```diff
@@ -75,10 +75,10 @@
 
         {% block js %}{% endblock %}
 
-        <a class="skiplink button" href="#main" data-skiplink>{% trans 'Skip to main content' %}</a>
+        <a class="skiplink button" href="#main" data-controller="w-skip-link" data-action="click->w-skip-link#skipToMain">{% trans 'Skip to main content' %}</a>
 
-        <div class="wrapper">
-            {% block furniture %}{% endblock %}
-        </div>
-    </body>
-</html>
+            <div class="wrapper">
+                {% block furniture %}{% endblock %}
+            </div>
+        </body>
+    </html>
```

Comment:
This change seems wrong - not sure what happened here but the div.wrapper should not have been indented.

Important: I think this will be a recurring problem with the HTML formatter @thibaudcolas  @Lovelyfin00  - Loveth - please raise a new issue about this in our call today. I think the `->` in the data-action is causing the formatter to think it is a closing block or similar.

For now we can work around this with using the shortcut syntax on the link
```
<a class="skiplink button" href="#main" data-controller="w-skip-link" data-action="w-skip-link#skip">{% trans 'Skip to main content' %}</a>
```